.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Now that you have this is the .github repo, why not just attach this task to the actions?

https://github.com/RoadieHQ/backstage-plugin-firebase-functions/actions/new

Reply:
Sorry, ignore me. I see this is actually what you have done.

Reply:
"Attaching this task to the actions" means commiting a workflow file. If you go to the link you provided and select some workflow template, the wizard will open a pull request with exact same file a in this pull request.
![image](https://user-images.githubusercontent.com/24685983/102610843-29c4da80-412e-11eb-8ebb-639741b1ae65.png)